### ISSUE TITLE
 predicted results of test dataset

### ISSUE THREAD

**Issue by DiuDiu-1123** [david-knigge/ccnn#13]
Through the code, I can only see test/acc and test/loss during testing. How can I see the predicted results of test dataset? And what kind of code should I add?

**Comment by dwromero** [comment]
Hi @DiuDiu-1123 
I am not sure I understand your question. Do you mean how to show the logits on the test dataset?

**Comment by DiuDiu-1123** [comment]
Yeah. I mean how to show the logits or the prediction on the test dataset. @dwromero
